packages/website/components/button.js

```diff
@@ -28,34 +28,23 @@ export default function Button({
   children,
   disabled = false,
 }) {
-  wrapperClassName = clsx(
-    'dib',
-    'bg-nsgray',
-    'ba',
-    'b-black',
-    { grow: !disabled, 'o-50': disabled },
-    wrapperClassName
-  )
-  const wrapperStyle = { minWidth: '8rem' }
-  const btnStyle = { top: 3, left: 3 }
+  const buttonStyle = { minWidth: '8rem', minHeight: '3.25rem' }
   const btn = (
     <button
       type={type}
       className={clsx(
-        'button-reset',
-        'relative',
-        'w-100',
-        'ba',
-        'b--black',
-        'pv2',
-        'ph3',
-        'chicagoflf',
-        'f5',
+        'w-full',
+        'border',
+        'border-black',
+        'rounded-md',
+        'px-4',
         { pointer: !disabled },
-        'bg-white',
+        'bg-black',
```

Comment:
The button doesn't have a hover effect now, you can add:
```css
hover:bg-gray-700
```

Reply:
I would say it's not worth adding it for now since the design is not finished yet. I will also need to add a light variant to the button, so it's better to deal with the hover states later on.